test/spec/utils/utils.ex

```diff
@@ -27,4 +27,15 @@ defmodule SpecTestUtils do
   defp parse_as_string(0), do: ""
   defp parse_as_string(x) when is_integer(x), do: :binary.encode_unsigned(x, :little)
   defp parse_as_string("0x" <> hash), do: Base.decode16!(hash, [{:case, :lower}])
+
+  def get_config("minimal"), do: MinimalConfig
+  def get_config("mainnet"), do: MainnetConfig
+
+  def read_ssz_from_file(file_path, ssz_type, config) do
+    with {:ok, compressed} <- File.read(file_path),
+         {:ok, decompressed} <- :snappyer.decompress(compressed),
+         {:ok, ssz_object} <- Ssz.from_ssz(decompressed, ssz_type, config) do
+      {:ok, ssz_object}
+    end
```

Comment:
Since this is a test util, I think we should use plain matches and just let it crash™
That way errors mention the specific line that caused the error.